consoleme/lib/aws.py

```diff
@@ -1827,7 +1826,7 @@ def allowed_to_sync_role(
     }  # Convert List[Dicts] to 1 Dict
 
     # All configured allowed_tags must exist in the role's actual_tags for this condition to pass
-    if allowed_tags and not allowed_tags.items() <= actual_tags.items():
-        allowed = False
+    if allowed_tags and allowed_tags.items() <= actual_tags.items():
```

Comment:
these changes are a duplicate of: https://github.com/Netflix/consoleme/pull/9213